src/components/hooks/index.tsx

```diff
@@ -0,0 +1,2 @@
+export {default as useMediaMatches} from "@/components/hooks/useMediaMatches";
+export {default as useWindowSize} from "@/components/hooks/useWindowSize";
```

Comment:
```suggestion
export * from "@/components/hooks/useMediaMatches";
export * from "@/components/hooks/useWindowSize";
```
defaultに名前を付けるのであれば、named export した方がコードが簡潔になるかと思います。